### PR TITLE
Add BlockExplorer-routes CAIP

### PR DESCRIPTION
**Pull request by ligi**
see discussion here: https://github.com/ChainAgnostic/CAIPs/discussions/199

### CODE REVIEW

File: CAIPs/caip-X.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+`<BLOCK_EXPORER_URL>/tx/<TX_HASH>`
+
+### Addresses
+
+`<BLOCK_EXPORER_URL>/address/<ACCOUNT_ADDRESS>`
+
+## Backwards Compatibility
+
+This EIP was designed with existing API routes in mind to reduce disruption. Incompatible block explorers should include either 301 redirects to their existing API routes to match this EIP.
```
Comment (bumblefudge) [inline CAIPs/caip-X.md:42]:
```suggestion
Incompatible block explorers should include either 301 redirects to their existing API routes to match this EIP.
```
Is there an "or XXX" missing here?

File: CAIPs/caip-X.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Explorers compatible with EIP-3091 are automatically comatible with this CAIP.
```
Comment (bumblefudge) [inline CAIPs/caip-X.md:43]:
```suggestion
Explorers compatible with EIP-3091 are automatically compatible with this CAIP.
```

**Comment by bumblefudge** [comment]
Forgive my ignorance here, but neither this CAIP nor the original EIP really explain how human-readable (i.e. web display) and machine-readable (API interactions properly speaking) interact here-- the [motivation section of the EIP](https://eips.ethereum.org/EIPS/eip-3091#motivation) refers to both wallets linking end-users to block explorer website AND API routes. It seems that the dapp<>wallet [RPC-based and CAIP-25-discovered] interaction of switching chains is supported by this, but how does the wallet know that the <block explorer URL> it's being asked to switch to will support these conformant routes?  I feel like wallets asked to switch to a previously un-registered chain would want to check a reputation registry/allowlist first, and this registry should also include up-to-date info about block explorer conformance/swappability, should this be spelled out maybe in a "Security Considerations" section?  (If I've gotten it close enough, happy to straw man such a section myself haha)

File: CAIPs/caip-X.md (hunk 1 above)
Comment (ligi) [reply CAIPs/caip-X.md:42]: yea - there is something missing actually - but same in the EIP - cc https://github.com/pedrouid

Comment (ligi) [reply CAIPs/caip-X.md:42]: but I think we can keep the sentence "This EIP was designed with existing API routes in mind to reduce disruption. " - just change EIP to CAIP

File: CAIPs/caip-X.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+--
+caip: <to be assigned>
```
Comment (ligi) [inline CAIPs/caip-X.md:2]:
```suggestion
caip: 200
```

File: CAIPs/caip-X.md (hunk 1 above)
Comment (ligi) [inline CAIPs/caip-X.md:42]:
```suggestion
This CAIP was designed with existing API routes in mind to reduce disruption. Incompatible block explorers should include either 301 redirects to their existing API routes to match this EIP.
```

File: CAIPs/caip-X.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+`<BLOCK_EXPORER_URL>/block/<BLOCK_HASH_OR_HEIGHT>`
+
+### Transactions
+
+`<BLOCK_EXPORER_URL>/tx/<TX_HASH>`
+
+### Addresses
+
+`<BLOCK_EXPORER_URL>/address/<ACCOUNT_ADDRESS>`
```
Comment (oed) [inline CAIPs/caip-X.md]: What's `ACCOUNT_ADDRESS` here? Would it make sense to rely on CAIP-10?

Comment (ligi) [reply CAIPs/caip-X.md]: would be nice - but then no explorer would be compatible and guess this CAIP would not get traction
But would really be nice for multi-chain explorers (are there any yet btw - did not yet stumble upon them)

File: CAIPs/caip-200.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+This is a CAIP with minimal routes that are used most of the time. If other routes are used in the future other CAIPs can inherit from this one and extend it. This way we can also differentiate between CAIPs with different feature-sets.
+
+## Specification
+
+Block explorers will route their webpages accordingly for the following data:
+
+### Blocks
+
+`<BLOCK_EXPORER_URL>/block/<BLOCK_HASH_OR_HEIGHT>`
+
```
Comment (ligi) [inline CAIPs/caip-200.md:31]:
```suggestion
```

File: CAIPs/caip-200.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+### Transactions
+
+`<BLOCK_EXPORER_URL>/tx/<TX_HASH>`
+
+### Addresses
+
+`<BLOCK_EXPORER_URL>/address/<ACCOUNT_ADDRESS>`
```
Comment (ligi) [inline CAIPs/caip-200.md:38]:
```suggestion
`<BLOCK_EXPLORER_URL>/address/<ACTOR_ADDRESS>`
```

**Comment by obstropolos** [comment]
Hey @ligi - pinging here for any additional review before getting this all wrapped up.

**Comment by ligi** [comment]
I think this is good to merge

**Pull request merged**